code/style.css

```diff
@@ -37,8 +121,8 @@ input {
   box-sizing: border-box;
   border: none;
   border-radius: 4px 0 0 4px;
-  background: #e5e9ff;
-  color: #0026ff;
+  background: #747474;
```

Comment:
Nice adding background color on input field. Maybe if you had the time - the bubbles also could have that grey color?

Reply:
Thanks for designing tips! If you have more tips please shoot me :)